utility/src/main/java/org/oppia/util/parser/CustomBulletSpan.kt

```diff
@@ -4,20 +4,24 @@ import android.content.Context
 import android.graphics.Canvas
 import android.graphics.Paint
 import android.graphics.Path
-import android.graphics.Path.Direction
 import android.text.Layout
 import android.text.Spanned
 import android.text.style.LeadingMarginSpan
 import org.oppia.util.R
 
-// TODO(#562): Add screenshot tests to check whether the drawing logic works correctly on all devices.
```

Comment:
Shouldn't we keep this TODO? We don't have screenshot testing support yet.

Reply:
Reverted this TODO comment.